lib/go_script/go.rb

```diff
@@ -90,10 +90,11 @@ def build_jekyll(extra_args = '')
   end
 
   def git_sync_and_deploy(commands, branch: nil)
-    exec_cmd 'git stash'
-    exec_cmd "git checkout -b #{branch}" unless branch.nil?
-    exec_cmd 'git pull'
-    exec_cmd 'bundle install' if File.exist? 'Gemfile'
+    exec_cmd 'git fetch origin #{branch}'
+    exec_cmd 'git clean -f'
+    exec_cmd "git reset --hard #{branch.nil? ? 'HEAD' : 'origin/' + branch}"
+    exec_cmd 'git submodule --sync --recursive'
+    exec_cmd 'git submodule update --init --recursive'
```

Comment:
Right, which will help keeps things stable as opposed to just pulling in the latest changes in any submodules without having vetted them first.  Thanks for verifying! :-)